.github/workflows/build.yml

```diff
@@ -31,3 +31,21 @@ jobs:
           ANACONDA_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
         run: |
           ./build.sh --include-user-channel --token "$ANACONDA_TOKEN" --push
+
+  build-macos-conda-packages:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: gadgetron-conda-recipes
+          environment-file: environment.yml
+          python-version: 3.9
+          auto-activate-base: false
+      - name: Build macos conda packages
+        shell: bash -l {0}
+        if: ${{ github.event_name == 'pull_request' }}
+        env:
```

Comment:
We actually don't need this ANACONDA_TOKEN here, since we are not pushing. But we will for the CI build (push). We can remove the token from the same step in Linux too.

Reply:
Removed in my latest commit to my branch (for both Linux and macOS builds).

Reply:
Confirmed this worked for macOS, and no reason it won't for Linux.